app/office/models.py

```diff
@@ -0,0 +1,64 @@
+from django.apps import apps
+from django.conf import settings
+from django.db import models
+from django.db.models.signals import post_delete, post_save
+from django.dispatch import receiver
+from django.utils import timezone
+from django_cleanup import cleanup
+from pathlib import Path
+from storages.backends.s3boto3 import S3Boto3Storage
+
+PATH = apps.get_app_config("office").path
+
+
+class ClosureTemplate(models.Model):
+    def _get_call_default():
+        return Path(PATH + "/closure/templates/call_template.txt").read_text()
+
+    def _get_email_default():
+        return Path(PATH + "/closure/templates/email_template.html").read_text()
+
+    def _get_notice_default():
+        return Path(PATH + "/closure/templates/notice_template.html").read_text()
```

Comment:
use render to string for templates - more idiomatic in Django

https://docs.djangoproject.com/en/5.0/topics/templates/#django.template.loader.render_to_string

Reply:
small nitpick but for Django models the methods tend to go below the field declarations